pkg/clientinteractor/interactor.go

```diff
@@ -3,6 +3,7 @@ package clientinteractor
 import (
 	"context"
 	"fmt"
+	"github.com/pg-sharding/spqr/pkg/models/keyspaces"
```

Comment:
We also need some 'default' key space and make keyspaces unseen for newbies. I'll explain later why.